ethcontract/src/tokens.rs

```diff
@@ -51,7 +52,9 @@ pub trait Tokenize {
 
 /// Wrapper around Vec<u8> and [u8; N] representing Token::{Bytes, FixedBytes}. Distinguishes a list
 /// of u8 from bytes.
-#[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[derive(
+    Clone, Copy, Debug, Default, Deserialize, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize,
+)]
```

Comment:
Does this need [`serde(transparent)`](https://serde.rs/container-attrs.html#transparent)?

Reply:
I don't think this "needs" anything because it isn't serde::Seralized by ethcontract or web3. This is because for users of ethcontract it is convenient to be able to derive Serlialize on event structs like we did in v1 to store them on disk.

Reply:
In my update web3 PR to services I had to remove the Serialize derive from the build script because of this (but also we don't need it to be serializable atm).

Reply:
Good point. I thought it needed to deserialize some `web3` related JSON. My only concern is that it conflicts a bit with the [`web3::types::Bytes` serialization implementation](https://docs.rs/web3/0.16.0/src/web3/types/bytes.rs.html#17-64) which might surprise users.

Reply:
Alternatively we could just have it not be serializable and if users want to serialize an event struct they'll have to create their own wrapper. For example if we didn't already depend on serde in this crate I wouldn't have liked to add the dependency just to make this Serializable.

Reply:
We do because `web3` and `jsonrpc-core` both depend on `serde`.

Reply:
So what do you think? Serializable or not?